composer.json

```diff
@@ -7,23 +7,46 @@
 		"email": "jason@stallin.gs"
 	}],
 	"require": {
-		"squizlabs/php_codesniffer": "2.8.*",
-		"wimg/php-compatibility": "dev-master",
+		"squizlabs/php_codesniffer": "2.9.*",
```

Comment:
Why stay on PHPCS 2.x when there is a separate composer config for PHP 5.2 ? Why not use the latest version which includes numerous improvements, including things which can not be worked around/back-filled in PHPCompatibility ?

Reply:
As far as I can tell PHPCS 3.x requires PHP 5.4, so we would need yet another sub-setup. The decision at this point was to stay with 2.x for that reason, to not overcomplicate further. That may be another iteration though.

Reply:
Understood. Just realize that there will be false positives caused by bugs in the PHPCS 2.x tokenizer which have been fixed in PHPCS 3.x, including some which were only fixed in PHPCS 3.2.0,

Reply:
Also a good point, but let's talk about this in an issue.  I'll create issues when merging.